pkg/network/tracer/tracer.go

```diff
@@ -290,6 +286,25 @@ func NewTracer(config *config.Config) (*Tracer, error) {
 	return tr, nil
 }
 
+func newConntracker(cfg *config.Config, conntrackerCreator func(*config.Config) (netlink.Conntracker, error)) (netlink.Conntracker, error) {
+	conntracker := netlink.NewNoOpConntracker()
+	if !cfg.EnableConntrack {
+		return conntracker, nil
+	}
+
+	if c, err := conntrackerCreator(cfg); err != nil {
+		if cfg.IgnoreConntrackInitFailure {
+			log.Warnf("could not initialize conntrack, tracer will continue without NAT tracking: %s", err)
+		} else {
+			return nil, fmt.Errorf("could not initialize conntrack: %s", err)
```

Comment:
nit: let's have change this error, or add logging to  print the configuration to bypass this somewhere